node-chrome/puppeteer-chrome-test.js

```diff
@@ -0,0 +1,65 @@
+const Apify = require('apify');
```

Comment:
Just please put a comment at the beginning describing what this does

Reply:
I'll do this, adding few more changes to the file.